external_search.py

```diff
@@ -381,15 +434,18 @@ def create_search_doc(self, query_string, filter, pagination,
             search = search.extra(explain=True)
 
         fields = None
-        if debug or return_raw_results:
+        if debug:
             # Don't restrict the fields at all -- get everything.
             # This makes it easy to investigate everything about the
             # results we do get.
-            fields = None
+            fields = ['*']
```

Comment:
Setting this to `None` was including all the regular fields but there was no way to add script fields. This way we get all the regular fields, plis we have a list to which to append the script fields.